build.sh

```diff
@@ -0,0 +1,51 @@
+name="oval-graph"
+module="oval_graph"
+
+git checkout master
+
+# Edit version, change log, etc... 
+vim ${module}/version.py
```

Comment:
Use `sed` to substitute version numbers automatically. For that, you need version.
Something like `sed -i /version=.*/version=$version/ file_with_version` could do the trick.